test/file_system_test.exs

```diff
@@ -4,8 +4,9 @@ defmodule FileSystemTest do
   @moduletag os_linux: true, os_macos: true, os_windows: true
 
   test "file event api" do
-    tmp_dir = System.cmd("mktemp", ["-d"]) |> elem(0) |> String.trim()
```

Comment:
AFAIK this command does not normally come with Windows and was absent on my machine.

This is not the greatest mktemp_d implementation but should be good enough.